Makefile

```diff
@@ -1,7 +1,16 @@
-short_ver = 2.0.2
+short_ver = 2.0.3
 long_ver = $(shell git describe --long 2>/dev/null || echo $(short_ver)-0-unknown-g`git describe --always`)
 generated = pglookout/version.py
 
+# Only include files that have been typed.
+typed = pglookout/__main__.py
```

Comment:
It could be a good idea to create a `mypy.ini` instead. That way we can use it to progressively type the library.
Something very simple à la https://github.com/aiven/pghoard/blob/main/mypy.ini

Reply:
@aris-aiven 
Done, I moved everything to `pyproject.toml`! 🌟